Gemfile

```diff
@@ -48,7 +48,7 @@ group :development do
   # gem 'rack-mini-profiler', '~> 2.0'
   gem 'listen', '~> 3.3'
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
-  # gem 'spring'
+  gem 'spring'
```

Comment:
what is spring? why did you uncoment?

Reply:
Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring